crates/pow/README.md

```diff
@@ -38,12 +38,19 @@ A challenge solution is valid when:
 
 ## Dynamic Difficulty
 
-The system automatically adjusts challenge difficulty based on usage:
-- **Target calculation**: `max_target / difficulty`
-- **Max target**: `u64::MAX >> baseline`
-- **Difficulty**: `max(num_active_challenges << growth_rate, 1)`
+The system automatically adjusts challenges difficulty based on usage and the request complexity. 
 
-This means as more users solve challenges, the difficulty increases exponentially, providing automatic rate limiting.
+The challenge target $t$ is computed as follows: $$ t = \dfrac{2^{64}} {d} $$
+
+We call $d$ "difficulty" such that $log_2(d)$ is the number of bits of work that we need to do to solve the challenge. We compute it as: $$ d = 2^b \centerdot c \centerdot n \centerdot g $$
```

Comment:
The formulas don't render for me properly. I think they need to be either in single `$` or if we use `$$`, then they need to be in separate code blocks. For example:

```
$$
d = 2^b \centerdot c \centerdot n \centerdot g
$$
```